roles/kube_prometheus_stack/defaults/main.yml

```diff
@@ -1,172 +1,42 @@
-
-# Release information for the NGINX ingress controller release
-kube_prometheus_release_namespace: monitoring-system
-kube_prometheus_release_name: kube-prometheus-stack
-
-# The timeout to wait for NGINX ingress controller to become ready
-# kube_prometheus_wait_timeout: 10m
-
-# Enable alerting on the Azimuth management cluster
-azimuth_management_cluster_alerting_enabled: false
-
-# Enable Slack alertmanager alerts
-kube_prometheus_stack_alertmanager_slack_alerts: false
-
-# Slack webhook URL for alertmanager alerts
-kube_prometheus_stack_alertmanager_slack_url: >- 
-  {{-
-    undef(hint = 'kube_prometheus_stack_alertmanager_slack_url is required')
-    if kube_prometheus_stack_alertmanager_slack_alerts
-    else None
-  }}
-
-# Slack channel for alertmanager alerts
-kube_prometheus_stack_alertmanager_slack_channel: >- 
-  {{-
-    undef(hint = 'kube_prometheus_stack_alertmanager_slack_channel is required')
-    if kube_prometheus_stack_alertmanager_slack_alerts
-    else None
-  }}
-
-# Size of the PVC for the prometheus database
-kube_prometheus_stack_prometheus_spec_volume_gb: 10
-
-# Max size of the prometheus database
-kube_prometheus_stack_prometheus_spec_retention_size: "{{ ( 0.95 * kube_prometheus_stack_prometheus_spec_volume_gb ) ~ 'GB' }}"
-
-# Spec for Prometheus pods
-kube_prometheus_stack_prometheus_spec: >-
-  {{-
-    {
-      "retentionSize": kube_prometheus_stack_prometheus_spec_retention_size,
-      "storageSpec": {
-        "volumeClaimTemplate": {
-          "spec": {
-            "accessModes": ["ReadWriteOnce"],
-            "resources": {
-              "requests": {
-                "storage": kube_prometheus_stack_prometheus_spec_volume_gb ~ 'Gi'
-                }
-              }
-            }
-          }
-        }
-      } 
-    }}
-
-# Default values for Prometheus section of kube-prometheus-stack
-kube_prometheus_stack_prometheus_values_defaults: >-
-  {{-
-    {
-    "prometheusSpec": kube_prometheus_stack_prometheus_spec
-    }
-  }}
-
-# Overrides for Prometheus section of kube-prometheus-stack
-kube_prometheus_stack_prometheus_values_overrides: {}
-
-# Values to pass to Prometheus part of kube-prometheus-stack
-kube_prometheus_stack_prometheus_values: >-
-  {{-
-      kube_prometheus_stack_prometheus_values_defaults |
-        combine(
-          kube_prometheus_stack_prometheus_values_overrides,
-          recursive = True
-        )
+kube_prometheus_stack_chart_repo: https://prometheus-community.github.io/helm-charts
+kube_prometheus_stack_chart_name: kube-prometheus-stack
+kube_prometheus_stack_chart_version: 45.4.0
+
+# Release information for the release
+kube_prometheus_stack_release_namespace: monitoring-system
+kube_prometheus_stack_release_name: kube-prometheus-stack
+
+# The timeout to wait for the release to become ready
+kube_prometheus_stack_wait_timeout: 10m
+
+# The values for the kube-prometheus-stack release
+kube_prometheus_stack_release_defaults:
+  prometheus:
+    prometheusSpec:
```

Comment:
Do we want a persistent volume by default on single node deployments?

Reply:
My feeling is that single nodes are mostly for development. Having a persistent volume seems like overkill.

Reply:
We can always add one down the line if it turns out to be an issue.